### PR TITLE
Revamp the Our Community section with new branding and designs

### LINKED ISSUE #135: Update Our community section
As part of the Kedro rebrand, we need to update the `Our community` section of the website to look like this:

image

Here's the Figma file: https://www.figma.com/file/fJLMAWpBfh75loDtclpdXR/Kedro-Website?type=design&node-id=1601%3A16983&t=jD0WMhiXRO8seHFJ-1

**Acceptance criteria:**

- [x] The section should animate/scroll automatically, from right to left, without any user input
- [x] When the animation gets to the end, it should start again, though not change directions (e.g. repeat from the beginning)
- [x] The `Our community` text should stay in place. Only the logos are moving.

See these links for inspiration:

- https://dvc.org/
- https://dagster.io/

### PR DESCRIPTION
**Pull request by ravi-kumar-pilla**
## Description

Resolves #135
 
## Development notes

Changed Our community section from displaying as grid to an infinite carousel using the [design link](https://www.figma.com/file/fJLMAWpBfh75loDtclpdXR/Kedro-Website?type=design&node-id=1622-5870&t=DQ1Fq7rg4ClDGwAH-0).

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Added tests to cover my changes, where applicable

### CODE REVIEW

**Comment by tynandebold** [comment]
@astrojuanlu the deploy preview is coming online now.

File: modules/index/companies-using-kedro/companies-using-kedro.module.scss [hunk 1]
```diff
@@ -16,57 +16,57 @@
...
+.titleContainer {
+  white-space: nowrap;
 
-    padding: 0;
+  .title {
+    @include display-md;
+    line-height: 7rem;
+    font-weight: 500;
+    padding-right: 2em;
+    @include breakpoint(small) {
```
Comment (tynandebold) [inline modules/index/companies-using-kedro/companies-using-kedro.module.scss:31]: Try and have a blank line before `@` rules.

```suggestion
    padding-right: 2em;
    
    @include breakpoint(small) {
```

File: modules/index/companies-using-kedro/companies-using-kedro.module.scss [hunk 2]
```diff
@@ -16,57 +16,57 @@
...
-  padding: 0;
-
-  @include breakpoint(small) {
-    grid-template-columns: 1fr 1fr;
-  }
+.carouselWrapper {
+  overflow: hidden;
+  width: 100%;
+  display: flex;
+  position: relative;
```
Comment (tynandebold) [inline modules/index/companies-using-kedro/companies-using-kedro.module.scss:44]: I like to try and sort things (CSS styles, React props) alphabetically so it's easier for a reader to find what they're looking for.

```suggestion
  display: flex;
  overflow: hidden;
  position: relative;
  width: 100%;
```

File: modules/index/companies-using-kedro/companies-using-kedro.tsx [hunk 3]
```diff
@@ -8,19 +8,39 @@ export default function CompaniesUsingKedro() {
...
+                loading="eager" // for Safari
+                className={style.company}
+                key={company.name}
+                alt={`${company.name} logo`}
+                height="120px"
+                src={company.logo}
+                width="308px"
+              />
+            ))}
+          </div>
```
Comment (tynandebold) [inline modules/index/companies-using-kedro/companies-using-kedro.tsx:42]: This code is repeated above. Is that by design? If so, why?

File: package-lock.json
Comment (tynandebold) [inline package-lock.json:1]: Can you try to remove this from your PR? I'm not sure why there are so many changes in this file. Which version of Node are you using when you run `npm install`? Try and use `v16`.

**Review by tynandebold — CHANGES_REQUESTED** [review]
Wow, your first PR already! Amazing job!

I've left some comments, a couple around some coding styles we like to try and stick to on the team.

One thing I'm noticing when looking at the animation in Chrome is that it's a little laggy/jumpy. Are you seeing the same? Maybe there's a way we can smooth that out with a browser prefix. It's very smooth on Safari.

Another thing: we'll need to do something different with the layout on mobile. For now, I think you can stack the `Our community` text above the animating logos and center align everything, following the convention from the other components.

Lastly, try and use hyphens in your branch names and be a little more descriptive with your PR naming. We don't need to micim what we use for branch names. I've updated it so you can see what I mean as an example.

Overall, amazing work! Onwards and upwards!

**Review by tynandebold — APPROVED** [review]
Looks good!

One last thing you need to change is to add a `max-width` to the element with a class of `.outer`. Looking at the designs, the max width of the content shouldn't be greater that `1280px`. Is that right @Mackay031.

Happy to approve now and have you add that in before merging.

**Review by Huongg — APPROVED** [review]
Looking good to me too @ravi-kumar-pilla, amazing job 😄

**Pull request merged**